MBBSEmu.Tests/ExportedModules/Majorbbs/ncdate_Tests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class ncdate_Tests : ExportedModuleTestBase
+    {
+        private const int NCDATE_ORDINAL = 428;
+
+        [Theory]
+        [InlineData(20785, "09/17/20")]
+        [InlineData(5535, "12/31/90")]
+        [InlineData(33, "01/01/80")]
```

Comment:
would be interesting to see what passing 0/-1 here does, as that's an invalid date

Reply:
Verified and it just returns a jacked up date -- formatted valid, but the values are just wrong.